camel/toolkits/function_tool.py

```diff
@@ -529,6 +544,86 @@ def generate_openai_tool_schema(
 
         return {}
 
+    def synthesize_output(self, args: Optional[Dict[str, Any]] = None) -> Any:
+        """
+        Synthesizes the execution output of the function.
+
+        Generates the output of the function based on the provided arguments
+        and the synthesis mode. Uses an assistant model to perform synthesis
+        if specified.
+
+        Args:
+            args (Optional[Dict[str, Any]]): Arguments to pass to the function
+                during synthesis. Defaults to `None`.
+
+        Returns:
+            Any: Synthesized output from the function execution. If no
+            synthesis model is provided, a warning is logged.
+        """
+        function_string = inspect.getsource(self.func)
+        if self.func.__doc__ is not None:
+            function_string += f"\n{self.func.__doc__}"
+        if args is not None:
+            function_string += f"\n{args}"
```

Comment:
How this aligns with the prompt?

Reply:
In my understanding, `args` will be used as part of the provided description. I tested it with GPT-4o and GPT-4o-mini, and it worked fine. But yes, I can improve the prompt to make this instruction clearer.